src/components/ShareLink.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import React, { PureComponent } from "react";
+import { StyleSheet, Share } from "react-native";
+import Icon from "react-native-vector-icons/dist/Feather";
+import Touchable from "./Touchable";
+import LText from "./LText";
+import colors from "../colors";
+
+type Props = {
+  style?: *,
+  children: string | React$Element<*>,
+  string: string, // String to be shared
+};
+
+class ShareLink extends PureComponent<Props> {
+  onPress = async () => {
+    const { string } = this.props;
+    await Share.share({
+      message: string,
+    });
+  };
+
+  render() {
+    const { style, children } = this.props;
+
+    return (
+      <Touchable
+        event="ShareLink"
+        style={[styles.linkContainer, style]}
```

Comment:
remove the `style` prop. it's not needed from the user side so no need to add this complexity

can just `style={styles.linkContainer}`